src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -216,6 +216,8 @@ public bool ReturnsByRefReadonly
 
         public abstract FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations { get; }
 
+        public abstract IImmutableSet<string> ReturnNotNullIfParameterNotNull { get; }
```

Comment:
>IImmutableSet [](start = 24, length = 13)

Are we relying on the interface or should this be `ImmutableHashSet<string>` instead?